ngraph/test/runtime/interpreter/reference/ctc_loss.hpp

```diff
@@ -4,8 +4,8 @@
 
 #pragma once
```

Comment:
CTC Loss reference implementation was moved to the [library with reference implementations](https://github.com/openvinotoolkit/openvino/tree/master/ngraph/core/reference). Please resolve merge conflicts

Reply:
Resolved